dashboards-observability/server/adaptors/custom_panels/custom_panel_adaptor.ts

```diff
@@ -119,6 +119,7 @@ export class CustomPanelsAdaptor {
         query: '',
         language: 'ppl',
       },
+      applicationId: appId || '',
```

Comment:
Are we keeping this field optional? or the value of the field optional? If the latter, then all the new-panels that are created from panels-home will have an applicationId field which is empty.

Reply:
I was thinking of using both. Panels that have already been created will not have the field at all, which I will account for when fetching panels and filtering out the ones from app analytics. The new panels will have an empty string for the applicationId. But I guess it is a bit more consistent to not have the field at all for operational panels created outside of app analytics. Typescript makes it look ugly but let me change it to the field being optional.